doc-content/enterprise-only/release-notes/rn-known-issues-ref-stash.adoc

```diff
@@ -0,0 +1,263 @@
+----
+
+Steps to reproduce:
+
+. Open {CENTRAL}.
+. Open the *Admin* page.
+. Open *Groups*.
+. Try to create a new group.
+
+
+Expected result: The new group is created.
+
+Actual result: A dialog box with the listed exception appears.
+
+Workaround: Add the group in RH-SSO instead of adding it in {CENTRAL}.
+
+.Human task comments are not retained when a process instance is finished
+//[https://issues.jboss.org/browse/JBPM-8556[JBPM-8556]]
+
+Issue: Human task comments are retained and visible only while the respective process instance is running. When the process instance is completed or aborted, comments from human tasks disappear even though the tasks are still visible in {CENTRAL}.
+
+Steps to reproduce:
+
+. Start a process instance with two consecutive human tasks.
+. Comment on the first human task.
+. Claim, start, and complete the first task.
+. Check the log file to confirm that the comments are retained for the completed task.
+. Abort the process instance or complete the task preventing the instance from finishing.
+. Navigate to the log file for the first task again.
+
+Expected result: Data about the first task is available in the log file.
+
+Actual result: The first task is still available in {CENTRAL} but no data is available in the log file.
+
+Workaround: None.
+
+
+.When you create an asset, the package drop-down button does not appear immediately [https://issues.jboss.org/browse/RHPAM-2133[RHPAM-2133]]
+
+Issue: When you create an asset, the package drop-down button is invisible and an error about package selection appears.
+
+Steps to reproduce:
+
+. Create a project in {CENTRAL}.
+. Try to create a business process.
+
+Expected result: The package drop-down button is visible and you can create the business process.
+
+Actual result: The package drop-down button is initially invisible and it appears after several seconds.
+
+Workaround: Wait until the package drop-down button is visible or reopen the dialog window for asset creation.
+
+ifdef::PAM[]
+
+
+
+== Process designer
+
+.Greater than (>) and less than (<) symbols in data types break a process [https://issues.jboss.org/browse/RHPAM-2193[RHPAM-2193]]
+
+Issue: If you place greater than (>) and less than (<) symbols in a data type for a process variable,  you cannot reopen the process. The XML editor appears instead.
+
+
+Steps to reproduce:
+
+. Create a process and add the `list1:java.util.List<String>` process variable.
+. Save and reopen the process.
+
+Expected result: The process opens.
+
+Actual result: The process does not open.
+
+Workaround: None.
+
+
+.If a case task actor is invalid, the resulting BPMN file is also invalid [https://issues.jboss.org/browse/RHPAM-2240[RHPAM-2240]]
+
+Issue: The new process designer sometimes generates an invalid XML file . A user task with an actor assigned is generated as a a `potentialOwner` element in the XSD file.
```

Comment:
Is the white space before the '.' harmless here?

Reply:
Fixed